types/react-i18next/index.d.ts

```diff
@@ -1,10 +1,10 @@
-// Type definitions for react-i18next 4.6
+// Type definitions for react-i18next 7.0
```

Comment:
Could you back up the old version to a folder called v4 as described in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version)? This should at least mitigate the breaking change.

Reply:
Actually looks like PR #21881 is also updating the version. Best not duplicate effort.

Reply:
Fixed